.travis.yml

```diff
@@ -0,0 +1,68 @@
+sudo: false
+
+language: php
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+branches:
+  only:
+    - master
+
+matrix:
+  include:
+    - php: 5.6
+      env: WP_VERSION=latest
+    - php: 5.6
+      env: WP_TRAVISCI=phpcs
+    - php: 7.0
+      env: WP_VERSION=latest
+    - php: 7.0
+      env: WP_VERSION=nightly
+    - php: 7.1
+      env: WP_VERSION=latest
+    - php: 7.2
+      env: WP_VERSION=latest
+  fast_finish: true
+  allow_failures:
+    - php: 7.2
```

Comment:
Since `7.2` is the standard on Park and Broadway, we should not allow failures here.